crates/air_r_formatter/tests/specs/r/binary_expression.R.snap

```diff
@@ -527,7 +527,7 @@ base_version <-
 
 -----
 Indent style: Tab
-Indent width: 2
+Indent width: 4
```

Comment:
Ignore all the diffs here, nothing actually changes because the indent style was tab

In a follow up PR I'll move us over to 2 spaces as the default